### DISCUSSION TITLE
Including NPIs in the .txt file

### DISCUSSION THREAD

**Discussion by gentry8** [CMSgov/hospital-price-transparency#54] [Q&A] [answered]
Hello,
I was wondering whether it was considered, or can considered for a future iteration of the .txt file mandate, to include each hospital's NPI in the body of text for each included location of the file. 

An example below:

location-name: Example Hospital East
**location-npi: 123456789**
source-page-url: https://example.com/price-transparency
mrf-url: https://example.com/price-transparency/123456789_Example-Hospital-East_standardcharges.csv
contact-name: Jon Snow
contact-email: jsnow@example.com

The reason being that hospitals' names and ownership can change quite frequently, making the location-name sometimes unreliable, or at least difficult to keep track of year-over-year.
The EIN at the start of each file's name may help alleviate this to some extent, but generally EINs are less widely available to the public than NPIs.

**Comment by RidenourK** [comment, marked as answer]
@gentry8. Thank you for your suggestion that we consider future regulation changes to require hospitals to include NPIs in their .txt files.
